File: oci-frontend/libs/core/src/lib/estados/estadoActividad.ts
export interface estadoActividadModelo {
  id: number;
  descripcion: string;
}

export const estadosActividad: estadoActividadModelo[] = [
  { id: 1, descripcion: 'Incumplida' },
  { id: 2, descripcion: 'Terminada' },
  { id: 3, descripcion: 'Activa' },
  { id: 4, descripcion: 'Por vencerse' },
];

File: oci-backend/src/main/java/co/edu/unicauca/oci/backend/apirest/utils/EstadoActividad.java
package co.edu.unicauca.oci.backend.apirest.utils;

public enum EstadoActividad {
    INCUMPLIDA("Incumplida"),
    TERMINADA("Terminada"),
    ACTIVA("Activa"),
    POR_VENCERSE("Por vencerse");

	private String nombre;
	private EstadoActividad(String nombre){
		this.nombre = nombre;
	}

	public String getNombre() {
		return nombre;
	}

}